eclass/kernel-2.eclass

```diff
@@ -1589,7 +1581,7 @@ kernel-2_src_prepare() {
 # conpile headers or run deblob script
 
 kernel-2_src_compile() {
-	cd "${S}"
+	cd "${S}" || die
 	[[ ${ETYPE} == headers ]] && compile_headers
 
 	if [[ $K_DEBLOB_AVAILABLE == 1 ]] && use deblob ; then
```

Comment:
`${K_DEBLOB_AVAILABLE}` I think was what was meant here ..

Reply:
I did make that change.  It says this comment is on an outdated file.

Reply:
just noticed that ! :)